### DISCUSSION TITLE
When can we have chat-with-pdf option?

### DISCUSSION THREAD

**Discussion by OpinionMatter** [nerve-sparks/iris_android#80] [Q&A] [answered]
Many users have suggested this feature, when is it expected to be available?

**Comment by Dhruvanand24** [comment, marked as answer]
The "Chat-with-PDF" or "Chat-with-documents" feature is highly anticipated and has been suggested by many users (#80). We are actively working on incorporating this functionality and aim to make it available in upcoming updates. Stay tuned for announcements regarding its release!
